compiler/src/iree/compiler/Dialect/Stream/Transforms/test/encode_device_tensors_packing.mlir

```diff
@@ -0,0 +1,119 @@
+// RUN: iree-opt --split-input-file --iree-stream-encode-device-tensors --verify-diagnostics %s | FileCheck %s
+
+// Check that i3 load is expanded right now.
```

Comment:
```suggestion
// Ensures that a non-power-of-two type (i3) is expanded to a full byte
// because we don't currently do unaligned sub-byte packing.
```

(it's useful for people trying to diagnose failures to have a hint here instead of having to find the exact line of code that may be doing the unexpected behavior)